net/pfSense-pkg-WireGuard/files/usr/local/pkg/wireguard/wg_guiconfig.inc

```diff
@@ -20,7 +20,7 @@
  */
 
 // pfSense includes
-require_once('guiconfig.inc');
+//require_once('guiconfig.inc');
```

Comment:
yeah I understand what it is doing. I was working on digging into a problem with one of my functions in `wg_guiconfig.inc`. Because every wg presentation page already required `guiconfig.inc`, having it required here was redundant and made it difficult to work on when `wg_guiconfig.inc` would also spew the `guiconfig.inc` bits when included by itself.

Reply:
So long as all the web-accessible files include `guiconfig.inc` themselves that is OK.